client/signup/steps/plans/index.jsx

```diff
@@ -27,8 +27,18 @@ import { getSiteBySlug } from 'calypso/state/sites/selectors';
 import './style.scss';
 
 export class PlansStep extends Component {
+	state = {
+		isDesktop: isDesktop(),
+	};
+
 	componentDidMount() {
-		this.props.saveSignupStep( { stepName: this.props.stepName } );
```

Comment:
Why do we need to remove this?

Reply:
Whoops, we don't! I've restored it in b09e78c60f